src/components/task-plan/homework.cjsx

```diff
@@ -184,6 +198,26 @@ AddExercises = React.createClass
 
 
 ExerciseSummary = React.createClass
+  componentDidMount: () ->
+    el = @getDOMNode()
+    @staticPosition = el.getBoundingClientRect().top - document.body.getBoundingClientRect().top
+    @handleScroll() # Update scroll position immediately on mount
+    @optimizedScrollFunc = throttle('scroll', 'optimizedScroll')
```

Comment:
why not just define this on the component directly? `optimizedScrollFunc: -> throttle('scroll', 'optimizedScroll')`